src/porepy/__init__.py

```diff
@@ -62,9 +62,10 @@
     constrain_geometry,
     map_geometry,
     geometry_property_checks,
-    point_in_polyhedron_test,
+    point_in_polyhedron,
```

Comment:
How the heck did this work previously?

Reply:
The file was renamed as well, so it all made sense, sort of.